DesyncSafeAnalyzer/DesyncSafeAnalyzer/Analyzers/PropertyAccessAnalyzer.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace DesyncSafeAnalyzer.Analyzers
+{
+  [DiagnosticAnalyzer(LanguageNames.CSharp)]
+  public class PropertyAccessAnalyzer : DiagnosticAnalyzer
```

Comment:
Currently this only prevents setting properties from DesyncSafe functions. It should also prevent setting properties from lambda functions passed into UnsyncUtils.InvokeForClient.